src/pages/components/avatar.mdx

```diff
@@ -53,8 +53,7 @@ import AvatarTypeCode from '!!raw-loader!../../examples/avatar/AvatarType.tsx';
 
 ### Status
 
-A status ring can be applied to the outside of an Avatar to communicate a user is away, available,
-or active.
+A colored ring outside the Avatar indicates a user's status. The options are away, available, or active.
```

Comment:
re: "user's"... What is the opinion re: https://en.wikipedia.org/wiki/Apostrophe – straight or curly? In the same vein, should we be checking for balanced quotes or other rules? If so, we might want to incorporate a tool like https://textlint.github.io/ into the website build.

Reply:
> How will the single-word "How to use it" headings scale to other pages? Avatar is a simple page. Consider a more complex page such as Inputs where example headings need to avoid conflicting with API component names.

You're referring to those those single-word subheadings like "Default" "Size" "Disabled" etc? If so, I'd first want to know how we think of those. Are those "features" of the component? Settings?

Reply:
Above I am talking about the single word headings you listed. I'll let @rossmoody answer to how we think about them.

Reply:
@vsri1137 I'd also like to hear your thoughts on the single vs. curly quote situation. We don't need to hold-up this PR, but there are automated linting solutions we can introduce to auto-fix some of these nits in the future if desired.